packages-next/keystone/src/lib/createImagesContext.ts

```diff
@@ -101,13 +103,13 @@ export function createImagesContext(config: KeystoneConfig['images']): ImagesCon
       if (isLocal(mode)) {
         const buffer = await fs.readFile(path.join(storagePath, `${id}.${extension}`));
         const metadata = await getImageMetadataFromBuffer(buffer);
-        const blurhash = await getBlurhashFromBuffer(buffer, metadata.width, metadata.height);
+        const blurHash = await getBlurhashFromBuffer(buffer, metadata.width, metadata.height);
```

Comment:
If we are gonna camel case then the function should be `getBlurHashFromBuffer` I guess

Reply:
No longer relevant as we are descoping blurhash